packages/expect/src/index.ts

```diff
@@ -416,22 +409,22 @@ function hasAssertions(...args: Array<any>) {
 }
 
 // add default jest matchers
-setMatchers(matchers, true, expect as Expect);
-setMatchers(spyMatchers, true, expect as Expect);
-setMatchers(toThrowMatchers, true, expect as Expect);
+setMatchers(matchers, true, expect);
+setMatchers(spyMatchers, true, expect);
+setMatchers(toThrowMatchers, true, expect);
 
-expect.addSnapshotSerializer = () => void 0;
```

Comment:
Is it breaking to remove this line? Could also be left in place with `// @ts-expect-error`.

Reply:
breaking, yes